examples/jsonpath-expressions/jsonpath_expressions.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/data.jsondata;
+import ballerina/io;
+
+public function main() returns error? {
+    json books = [
```

Comment:
Shall we define this as module level variable?

Reply:
Update like this according to https://github.com/ballerina-platform/ballerina-distribution/pull/5287#discussion_r1557303461